src/python/grpcio_test/grpc_test/framework/common/test_control.py

```diff
@@ -34,6 +34,10 @@
 import threading
 
 
+class Defect(Exception):
+  """Simulates a programming defect in a system under test."""
```

Comment:
Is this done specifically to make the raised error class a non-standard one? If so, mind adding the relevant tidbits of the commit message to this comment?